### PR TITLE
Migrate the ecobalyse-data repository contents into src/.

### PR DESCRIPTION
**Pull request by magopian**
Le but est de fusionner le repo [ecobalyse-data](https://github.com/MTES-MCT/ecobalyse-data) directement dans `src/` afin d'éviter les soucis liés à un multi-repo.

Nous allons conserver le repo de ecobalyse-data pour des scripts qui ne sont pas directement utiles à ecobalyse.

### CODE REVIEW

**Review by n1k0 — COMMENTED** [review]
*Disclaimer: debatable personal feelings and opinions about things.*

Personally, I'd prefer the Python codebase to live elsewhere than `/src`, eg in `/data` at the root of the repo, mainly because imho the `src` folder — being "owned" by `elm.json` — is really intended to contain Elm and only Elm code; putting Python, JSON or CSV stuff in there is counter intuitive ***to me***.

I'd like to have opinions of both @paulboosz and @ccomb about this before merging this patch.

**Comment by magopian** [comment]
What you said makes sense to me too, I don't see anything preventing us from moving `src/ecobalyse-data/` to `ecobalyse-data/` or even `data/`.

File: src/ecobalyse_data/food/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+forme de [notebooks jupyter](https://jupyter.org/) pour explorer et expérimenter
+par vous-même.
+
+## Installer anaconda et créer son environnement virtuel
+
+Suivre la [procédure d'installation](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+Créer un environnement virtuel avec toutes les dépendences requises :
+
+    $ conda env create -f food/environment.yml
```
Comment (n1k0) [inline src/ecobalyse_data/food/README.md:30]: Argh, now I see that instructions were there too. THat should probably be stated in the README at the root of the repo⋅⋅⋅

**Review by n1k0 — COMMENTED** [review]
So I was reviewing all these files and then I suddenly realized that maybe we want to land everything and reorganize things after the patch being merged? Or do we want to clean as much stuff as possible before landing? I'd personally vote for the latter but I'd like to know what @ccomb and @paulboosz think?

**Comment by n1k0** [comment]
Yes, totally a possibility, I'd just really love to have clear instructions in the README at the root of the repo about what the different commands I can run to generate the exports. Right now I'm still crawling READMEs and toying around calls to conda and python scripts to achieve that, with no success so far. I think that would be a bare minimum before actually landing the patch, but that may just be me!

**Comment by magopian** [comment]
Ok I just pushed a new commit with (hopefully) improved READMEs, do they help @n1k0 ?

File: src/ecobalyse_data/food/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Importer la base agribalyse v3.0 dans brightway
+
+Tout d'abord se placer dans le bon répertoire :
+
+    $ cd import_agb
+
+Attention : il est nécessaire d'avoir le fichier `agribalyse3_no_param.CSV` dans
+le répertoire. Ce fichier est un export de la base Agribalyse par le biais du
+logiciel (payant) SimaPro. Veuillez nous contacter si vous en avez besoin.
```
Comment (n1k0) [inline src/ecobalyse_data/food/README.md:32]: Actually the file needed is `agribalyse3_no_param.CSV.zip` (note the `.zip` extension). If you just put the csv file, you get this error:

```
Creating core data migrations

Traceback (most recent call last):
  File "/home/niko/work/akei/ecobalyse/src/ecobalyse_data/food/import_agb/importing_databases.py", line 108, in <module>
    main()
  File "/home/niko/work/akei/ecobalyse/src/ecobalyse_data/food/import_agb/importing_databases.py", line 46, in main
    with ZipFile(agb_csv_zip_filepath) as zf:
  File "/home/niko/miniconda3/envs/ecobalyse-env/lib/python3.9/zipfile.py", line 1248, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'agribalyse3_no_param.CSV.zip'
```

File: src/ecobalyse_data/food/export_agb/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+## Exporter les données pour le constructeur de recettes
+
+Lancer le script d'export :
+
+    $ python export_builder.py
+
+Exemple :
+
+    $ python export_builder.py
```
Comment (n1k0) [inline src/ecobalyse_data/food/export_agb/README.md:36]: These two command lines are the same.

Also, when running this I'm getting `AssertionError: Can't find method object ('EF v3.0', 'acidification', 'accumulated exceedance (ae)')`:

```
(ecobalyse-env) ➜  export_agb git:(merge-ecobalyse-data) python export_ciqual.py --max 1 --no-impacts
Get ciqual codes from ../Agribalyse_Synthese.csv
Open the agribalyse3 brightway database
Biosphere database already present!!! No setup is needed
Search for the ciqual products in the brightway database
Loaded 1 products
Building product tree
100%
Export de 1 produits vers ../../../../public/data/food/products.json
Export de 17 produits vers processes-no-impacts.json
Terminé.
(ecobalyse-env) ➜  export_agb git:(merge-ecobalyse-data) ✗ python checks_ciqual.py
>>> Liste des produits avec étapes manquantes
0 missing steps
(ecobalyse-env) ➜  export_agb git:(merge-ecobalyse-data) ✗ python export_builder.py
10 procédés à rajouter provenant de ingredients_base.json (ingrédients complexes)
Total de 54 procédés à exporter
Biosphere database already present!!! No setup is needed
Chargement de 54 activités terminé
Total de 54 activités trouvées dans agribalyse
Initialisation de la méthode ('EF v3.0', 'acidification', 'accumulated exceedance (ae)')
Traceback (most recent call last):
  File "/home/niko/work/akei/ecobalyse/src/ecobalyse_data/food/export_agb/export_builder.py", line 342, in <module>
    lcas = init_lcas({random_process: 1})
  File "/home/niko/work/akei/ecobalyse/src/ecobalyse_data/food/export_agb/export_builder.py", line 151, in init_lcas
    lca = bw.LCA(demand, method)
  File "/home/niko/miniconda3/envs/ecobalyse-env/lib/python3.9/site-packages/bw2calc/lca.py", line 106, in __init__
    self.get_array_filepaths()
  File "/home/niko/miniconda3/envs/ecobalyse-env/lib/python3.9/site-packages/bw2calc/lca.py", line 124, in get_array_filepaths
    get_filepaths(self.method, "method"),
  File "/home/niko/miniconda3/envs/ecobalyse-env/lib/python3.9/site-packages/bw2calc/utils.py", line 98, in get_filepaths
    assert name in metadata, "Can't find {} object {}".format(kind, name)
AssertionError: Can't find method object ('EF v3.0', 'acidification', 'accumulated exceedance (ae)')
```

**Review by n1k0 — COMMENTED** [review]
Here's some feedback when following the procedure. I'm stuck at the `export_builder.py` step.

File: src/ecobalyse_data/food/export_agb/README.md (hunk 3 above)
Comment (n1k0) [reply src/ecobalyse_data/food/export_agb/README.md:36]: Interesting comment from @ccomb over Mattermost regarding this error : 

> Et au passage j'ai la confirmation qu'avec bw2io=0.8.6 (ou 7) ça passe, et avec 0.8.8 ça refait l'erreur AssertionError: Can't find method object ('EF v3.0', 'acidification', 'accumulated exceedance (ae)')
> Donc pour l'instant je bloque en 0.8.7

We may want to ensure dependency versions are pinned.

**Comment by magopian** [comment]
@n1k0 I pinned the dependencies in `environment.yml` and updated the README to add the `.zip` extension. Does that work better for you?

**Comment by n1k0** [comment]
> I pinned the dependencies in environment.yml and updated the README to add the .zip extension. Does that work better for you?

I don't know, I've removed my tests as I needed to switch to another branch, and I'm off today. I trust you, if you feel the patch is in a good shape for landing, land it.

**Comment by n1k0** [comment]
Wait, could you please at least move `/src/ecobalyse-data` to just `/data`? Thx

**Review by n1k0 — APPROVED** [review]

**Pull request merged**